decidim-core/db/migrate/20220203121137_add_notifications_sending_frequency_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddNotificationsSendingFrequencyToUsers < ActiveRecord::Migration[6.0]
```

Comment:
I'd like to hear @ahukkanen take on this, but I think this should be migrated to the `notifications_settings` column, as the plan is to eventually migrate all these notification related columns to this new format. This would apply too to `digest_sent_at`.

Reply:
@andreslucena I would keep the `notification_settings` column only for the notification toggles, i.e. which notifications are enabled/disabled. For me this approach seems fine that we store the sending frequency in a separate column.